Gemfile

```diff
@@ -5,6 +5,7 @@ ruby '3.3.0'
 gem 'rails', '~> 7.1'
 gem 'rails-i18n'
 
+gem 'actioncable-enhanced-postgresql-adapter'
```

Comment:
Ak spravne pozeram, tak pod 8000 bytes sa to sprava rovnako ako default adapter, cize sem nemusime tahat zavislost na nejaky deravy mrtvy gem 2 roky stary. (partial co posielas mas < 2000 bytes)